data.c

```diff
@@ -516,12 +516,13 @@ backup_data_file(const char *from_root,
 		file->read_size += read_len;
 	}
 	/*
-	 * In incremental backup mode, put an additional data to remember
-	 * the last block number.
-	 * Without this, if a vacuum shrinks pages, which happens when one
-	 * or more pages at the end of a table are deleted, pg_rman has
-	 * no way to know those pages are shrinked and the deleted pages
-	 * comeback during pg_rman recovery.
+	 * In incremental backup mode, append a special 0-filled page, with
+	 * lastpage field set to true, to mark the end of relation as of this
+	 * backup.
+	 * This is used when restoring to truncate any subsequent pages that
```

Comment:
Sorry that I didn't add it to begin with, but please add an additional word and punctuation for clarity:

This is used when restoring the backup, to truncate any subsequent pages that